setup.py

```diff
@@ -1,27 +1,25 @@
 import os
 from setuptools import setup, find_packages
-version = '0.3.0'
-README = os.path.join(os.path.dirname(__file__), 'README')
+version = '1.0.0'
+README = os.path.join(os.path.dirname(__file__), 'README.md')
 long_description = open(README).read()
 setup(name='locking',
       version=version,
+      packages=find_packages(),
       description=("Prevents users from doing concurrent editing in Django. Works out of the box in the admin interface, or you can integrate it with your own apps using a public API."),
       long_description=long_description,
+      url='https://github.com/unt-libraries/django-locking',
+      author='University of North Texas Libraries',
+      author_email='mark.phillips@unt.edu',
+      license='BSD',
+      keywords='locking mutex',
       classifiers=['Development Status :: 4 - Beta',
                    'Environment :: Web Environment',
-                   'Framework :: Django',
+                   'Framework :: Django :: 2.2',
                    'Intended Audience :: Developers',
                    'License :: OSI Approved :: BSD License',
                    'Operating System :: OS Independent',
                    'Programming Language :: Python',
                    'Topic :: Software Development :: Libraries :: Python Modules',
                    'Topic :: Utilities'],
-      keywords='locking mutex',
-      author='Stijn Debrouwere',
-      author_email='stijn@stdout.be',
-      url='http://stdbrouw.github.com/django-locking/',
-      download_url='http://www.github.com/stdbrouw/django-locking/tarball/master',
-      license='BSD',
-      packages=find_packages(),
-      install_requires=['django-staticfiles','simplejson'],
```

Comment:
I am wondering if we should add `django` as a dependency here in `install_requires`.

Reply:
well...I think since in the README we are mentioning python and Django as system dependencies, it is expected to have them installed before using the app. would it be necessary to have it here?

Reply:
I wouldn't say it is necessary, but since we are instructing people to use a virtual environment, it would make sense to me they would install Django there and not as a system level dependency. We usually have django 2.2 in a test requirements file that we instruct people to install. Since we aren't using requirements files, I though tit might make sense to put it in the `install_requires`. What do you think @somexpert ?

Reply:
I don't have strong feelings about this, since I don't think I've been very consistent with it myself... We _do_ mention virtual environments in the development section, but not for the main installation. In either case, having it in `install_requires` shouldn't be an issue, so I say we do it and get rid of it from the README.